### PR TITLE
Edits to the readme

### PR DESCRIPTION
**Pull request by chucknado**
### Description

* Makes edits and fixes links to old docs

ADMIN: We need a repo admin to turn on Issues (under Settings). I don't have permissions. In the readme, we encourage people to report bugs with Github issues, but the feature isn't turned on.

### Reference
https://support.zendesk.com/agent/tickets/3042635

@zendesk/vegemite

### CODE REVIEW

**Comment by grosser** [comment]
turned issues on!

**Comment by grosser** [comment]
@chucknado can you ping someone for review ... nobody seems to care :D

**Comment by svizzari** [comment]
@chucknado 

> ADMIN: We need a repo admin to turn on Issues (under Settings). I don't have permissions. In the readme, we encourage people to report bugs with Github issues, but the feature isn't turned on.

Sorry, missed a couple of GH notifications in the past week, else I would have actioned this for you straight away.  Thanks @grosser 👏

File: README.md [hunk 1]
```diff
@@ -1,30 +1,40 @@
 # Zendesk Apps Tools
 
 ## Description
-Zendesk Apps Tools (ZAT) is the only tool needed for developing Zendesk Apps. It makes it easy to develop [Zendesk Apps](http://developer.zendesk.com/documentation/apps/). We have a [guide](http://developer.zendesk.com/documentation/apps/reference/tools.html) that explains how to use this tool.
```
Comment (danielbreves) [inline README.md]: lol... life was so much simpler back then :)

**Pull request merged**